docs/devtools.md

```diff
@@ -1,45 +1,68 @@
 # Firebase Genkit Developer Tools
 
-Firebase Genkit comes with two sets of developer tools:
+Firebase Genkit provides two key developer tools:
 
-- A Node.js CLI
-- An optonal local web app that can connect to your Genkit configuration
+- A Node.js CLI for command-line operations
+- An optional local web app, called the Developer UI, that interfaces with your Genkit configuration for interactive testing and development
 
-### Command Line Interface
+### Command Line Interface (CLI)
 
-Install the CLI with the following command:
+Install the CLI in your project using:
 
 ```posix-terminal
-npm install genkit
+npm install -D genkit-cli
 ```
 
-The CLI offers many useful commands to work with Genkit projects including:
+The CLI supports various commands to facilitate working with Genkit projects:
 
-- `genkit init`: initialize a Genkit project
-- `genkit flow:run flowName`: run a flow
-- `genkit eval:flow flowName`: evaluate a flow
+- `genkit flow:run <flowName>`: Run a specified flow.
+- `genkit eval:flow <flowName>`: Evaluate a specific flow.
```

Comment:
Maybe replace with `genkit start <command>: execute command and start the developer UI with a connected environment` since I think `genkit start` is the main command we expect people to use